lib/widgets/lightbox.dart

```diff
@@ -492,11 +493,25 @@ class _VideoLightboxPageState extends State<VideoLightboxPage> with PerAccountSt
     _controller?.removeListener(_handleVideoControllerUpdate);
     _controller?.dispose();
     _controller = null;
+    WakelockPlus.disable();
     super.dispose();
   }
 
-  void _handleVideoControllerUpdate() {
-    setState(() {});
+  Future<void> _handleVideoControllerUpdate() async {
+    if (mounted) setState(() {});
+
+    final wakelockEnabled = await WakelockPlus.enabled;
+    if (_controller!.value.isPlaying) {
```

Comment:
Could `_controller` be null here? I think maybe so, if the widget were disposed during the async gap. So we probably want an early return if `_controller == null`.